lua/lv-formatter/init.lua

```diff
@@ -11,8 +11,8 @@ if O.format_on_save then
   }
 end
 
--- check if formatter has been defined for the language or not
-local function formatter_exists(lang_formatter)
+-- check if formatter configuration is available inside `Formatter` in default-config.lua or not
+local function fmt_conf_present(lang_formatter)
```

Comment:
Since basically this is `return lang_formatter != nil`, remove the function and replace the calls with the condition

Reply:
it also checks for `exe` and `args` to be filled as well